src/sql/src/catalog.rs

```diff
@@ -62,6 +62,13 @@ pub trait Catalog: fmt::Debug {
     /// Returns a persistent UUID for the the catalog.
     fn cluster_id(&self) -> Uuid;
 
+    /// Returns the search path used by the catalog.
+    ///
+    /// If `include_system_schemas` is true, the `MZ_CATALOG`, `PG_CATALOG`,
```

Comment:
nit: this trait shouldn't have any opinion on what the concrete system schemas are. That responsibility is left to the trait's implementors.